chstrings/__init__.py

```diff
@@ -75,6 +75,53 @@ def _preprocess_variables(config, strings):
                 tooltitle = strings['tooltitle'],
                 days = '%s'))  # The template swaps in the actual number.
 
+    strings.setdefault('intersection_intro', '')
```

Comment:
We only really set a default value when the string requires some post-processing such as interpolating another string into it, or treating it as safe markup (the `flask.Markup` stuff). For other strings, we just leave them blank and take care not to use them later if they're not there.

I've seen other software just default to English, but I'm not a fan of that. In the long term, I think it would be pretty cool to automatically disable features until all of their required strings are there...

Reply:
Filed #123 to make this a bit smarter in the future.